templates/server/index.html

```diff
@@ -169,59 +159,114 @@ <h3 class="p-muted-heading u-align--center">Works with all your hardware and sof
+        </p>
+        <p>
+          <a class="p-link--external" href="https://maas.io/">
+            Learn more about MAAS
+          </a>
+        </p>
       </div>
     </div>
     <div class="col-4 p-divider__block">
       <div class="p-heading-icon">
         <div class="p-heading-icon__header">
           <img class="p-heading-icon__img" src="{{ ASSET_SERVER_URL }}425efe3a-lxd.svg" width="100" alt="LXD" />
-          <h3 class="p-heading-icon__title">LXD &mdash; system containers</h3>
+          <h3 class="p-heading-icon__title">
+            LXD &mdash; machine containers
+          </h3>
         </div>
-        <p>LXD, the Linux container hypervisor, merges the speed and density of containers with the manageability and security of traditional virtual machines. Economics are directly tied to density, and no other virtualisation technology is as fast or dense as LXD.</p>
-        <p><a class="p-link--external" href="https://linuxcontainers.org/">Learn more about LXD</a></p>
+        <p>
+          Economics are directly tied to compute density. LXD, the Linux container hypervisor, merges the speed and density of containers with the manageability and security of traditional virtual machines.
+        </p>
+        <p>
+          <a class="p-link--external" href="https://linuxcontainers.org/">
+            Learn more about LXD
+          </a>
+        </p>
       </div>
     </div>
   </div>
-</div>
+</section>
 
-<div class="p-strip--light is-bordered">
+<section class="p-strip is-bordered">
   <div class="row">
     <div class="col-12">
-      <h2>Ubuntu Advantage: World class support for your Infrastructure</h2>
+      <h2>
+        Ubuntu Advantage for your Infrastructure
+      </h2>
     </div>
   </div>
   <div class="row">
     <div class="col-6">
-      <p>The Ubuntu Advantage service programme provides fast problem resolution, direct access to Ubuntu experts and efficient administration with the Ubuntu systems management package, Landscape. Available in Standard (business hours) and Advanced (24x7) support.</p>
-      <p>UA for Infrastructure, Advanced includes Server, OpenStack and Kubernetes support.</p>
-      <p><a href="/support">Learn more about Ubuntu Advantage for Infrastructure&nbsp;&rsaquo;</a></p>
+      <p>
+        Ubuntu Advantage for infrastructure offers a single, per-node packaging of the most comprehensive software, security and IaaS support in the industry. With OpenStack and Kubernetes support included, Ubuntu Advantage for Infrastructure provides everything you need to future-proof your data centre.
+      </p>
+      <p>
+        UA for Infrastructure, Advanced includes Server, OpenStack and Kubernetes support. and is available at Standard (24x5) and Advanced (24x7) response times.
```

Comment:
Copy doc has "UA for Infrastructure, Advanced includes Server, OpenStack and Kubernetes support and is available at Standard (24x5) and Advanced (24x7) response times."